charts/redpanda/values.yaml

```diff
@@ -691,31 +691,47 @@ statefulset:
+    # * Every container in the Pod must have a CPU limit and a CPU request.
+    # * For every container in the Pod, the CPU limit must equal the CPU request.
+    #
+    # To maximize efficiency, use the `static` CPU manager policy by specifying an even integer for
+    # CPU resource requests and limits. This policy gives the Pods running Redpanda brokers
+    # access to exclusive CPUs on the node. For details, see
+    # https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy
+    resources: {}
+    securityContext: {}
+    extraVolumeMounts: ""
+    image:
+      tag: v2.3.5-24.3.2
+      repository: docker.redpanda.com/redpandadata/redpanda-operator
+    # The PVCUnbinder helps keep redpanda operational in the event of a tolerable Node or Disk loss event when non-remountable storage,
+    # such as `local` or `hostPath`, are used. It does so by monitoring redpanda Pods that are in a "Pending" state for at least the period
+    # specified by `unbindAfter`. After this period it will delete the Pod's PVC to re-trigger volume provisioning.
+    pvcUnbinder:
+      enabled: false
+      unbindAfter: 60s
+    # The BrokerDecommissioner attempts to decommission brokers that have been non-gracefully terminated in a cluster due to something
+    # like a Node crash. When a broker is determined to no longer be valid (i.e. it has been replaced by a broker with a different id)
+    # the decommissioner reaches out to the deployed cluster and begins a decommission operation after detecting the broker as invalid
+    # multiple times across the `decommissionAfter` window. A `decommissionRequeueTimeout` values can be specified to override how
+    # often the decommissioner rechecks a cluster that has a broker which has been flagged as invalid. 
+    brokerDecommissioner:
+      enabled: false
+      decommissionAfter: 60s
+      decommissionRequeueTimeout: 10s
     configWatcher:
       enabled: true
-      # -- To create `Guaranteed` Pods for Redpanda brokers, provide both requests and limits for CPU and memory. For details, see
-      # https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed
-      # * Every container in the Pod must have a memory limit and a memory request.
-      # * For every container in the Pod, the memory limit must equal the memory request.
-      # * Every container in the Pod must have a CPU limit and a CPU request.
-      # * For every container in the Pod, the CPU limit must equal the CPU request.
-      #
-      # To maximize efficiency, use the `static` CPU manager policy by specifying an even integer for
-      # CPU resource requests and limits. This policy gives the Pods running Redpanda brokers
-      # access to exclusive CPUs on the node. For details, see
-      # https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy
+      # deprecated
```

Comment:
NIT: Could you prefix word deprecated with `--` that way the README.md will indicate that it's deprecated

```suggestion
      # -- DEPRECATED: Please use statefulset.sideCars.resources
```

I follow what is done in 
https://github.com/redpanda-data/redpanda-operator/blob/054413d40605338549a4fd85786a817599d7fa19/charts/redpanda/values.yaml#L688

which is included by helm-docs cli
https://github.com/redpanda-data/redpanda-operator/blob/054413d40605338549a4fd85786a817599d7fa19/charts/redpanda/README.md?plain=1#L949-L951

https://github.com/redpanda-data/redpanda-operator/blob/main/charts/redpanda/README.md#statefulsetsecuritycontext